GitUI/Editor/FileViewer.cs

```diff
@@ -467,8 +467,7 @@ public void ViewStagingPatch(Patch patch)
 
         public void ViewPatch([CanBeNull] Patch patch)
         {
-            string text = patch != null ? patch.Text : "";
-            ViewPatch(text);
+            ViewPatch(patch?.Text ?? "");
```

Comment:
This fixes a potential NRE as `Text` can be `null` and `ViewPatch` doesn't accept `null`.